filemage/tests/test_filemage.py

```diff
@@ -0,0 +1,491 @@
+            return False
+        try:
+            return not (self != psutil.Process(self.pid))
+        except psutil.ZombieProcess:
+            # We should never get here as it's already handled in
+            # Process.__init__; here just for extra safety.
+            return True
+        except psutil.NoSuchProcess:
+            self._gone = True
+            return False
+
+    def cmdline(self):
+        return self._cmdline
+
+    def status(self):
+        return self._status
+
+    def ppid(self):
+        return self._ppid
+
+    def name(self):
+        return self._name
+
+    def cwd(self):
+        raise NotImplementedError()
+
+    def username(self):
+        raise NotImplementedError()
+
+    def exe(self):
+        raise NotImplementedError()
+
+    def create_time(self):
+        return self._create_time
+
+    def nice(self, value=None):
+        raise NotImplementedError()
+
+    def uids(self):
+        raise NotImplementedError()
+
+    def gids(self):
+        raise NotImplementedError()
+
+    def terminal(self):
+        raise NotImplementedError()
+
+    def num_fds(self):
+        raise NotImplementedError()
+
+    def io_counters(self):
+        raise NotImplementedError()
+
+    def ionice(self, ioclass=None, value=None):
+        raise NotImplementedError()
+
+    def rlimit(self, resource, limits=None):
+        raise NotImplementedError()
+
+    def cpu_affinity(self, cpus=None):
+        raise NotImplementedError()
+
+    def cpu_num(self):
+        raise NotImplementedError()
+
+    def environ(self):
+        raise NotImplementedError()
+
+    def num_handles(self):
+        raise NotImplementedError()
+
+    def num_ctx_switches(self):
+        raise NotImplementedError()
+
+    def num_threads(self):
+        raise NotImplementedError()
+
+    def threads(self):
+        raise NotImplementedError()
+
+    def children(self, recursive=False):
+        raise NotImplementedError()
+
+    def cpu_percent(self, interval=None):
+        raise NotImplementedError()
+
+    def cpu_times(self):
+        raise NotImplementedError()
+
+    def memory_info(self):
+        raise NotImplementedError()
+
+    def memory_full_info(self):
+        raise NotImplementedError()
+
+    def memory_percent(self, memtype="rss"):
+        raise NotImplementedError()
+
+    def memory_maps(self, grouped=True):
+        raise NotImplementedError()
+
+    def open_files(self):
+        raise NotImplementedError()
+
+    def connections(self, kind='inet'):
+        raise NotImplementedError()
+
+    def send_signal(self, sig):
+        raise NotImplementedError()
+
+    def suspend(self):
+        raise NotImplementedError()
+
+    def resume(self):
+        raise NotImplementedError()
+
+    def terminate(self):
+        raise NotImplementedError()
+
+    def kill(self):
+        raise NotImplementedError()
+
+    def wait(self, timeout=None):
+        raise NotImplementedError()
+
+
+class MockResponse:
```

Comment:
There is a `MockResponse` in `datadog_checks.dev.http` ([link](https://github.com/DataDog/integrations-core/blob/4978cc1fd8137e4b7bc84aae84bfee6a747ee6f3/datadog_checks_dev/datadog_checks/dev/http.py#L11)) ([example](https://github.com/DataDog/integrations-core/blob/4978cc1fd8137e4b7bc84aae84bfee6a747ee6f3/vault/tests/test_vault.py#L66-L84)), so you shouldn't need to roll out your own.